Resources/docs/index.rst

```diff
@@ -0,0 +1,378 @@
+Connector installation
+======================
+
+Add ConnectionsBundle and OXIDConnectorBundle to ``AppKernel.php``
+
+.. code-block:: php
+
+    new ONGR\ConnectionsBundle\ONGRConnectionsBundle(),
+    new ONGR\OXIDConnectorBundle\ONGROXIDConnectorBundle(),
+
+..
+
+Then in your bundle you will need to extend ``Documents`` and ``Entities`` provided in OXIDConnectorBundle
+in following locations:
+
+- ``OXIDConnectorBundle/Document`` for documents
+- ``OXIDConnectorBundle/Entity`` for entities
+
+Then you will need to add mappings for these documents and entities. Examples:
+
+.. code-block:: yaml
+
+    orm:
+        auto_generate_proxy_classes: "%kernel.debug%"
+        auto_mapping: true
+        mappings:
+            ONGROXIDConnectorBundle:
+                type: annotation
+                alias: OXIDConnectorBundle
+                dir: Entity
+                prefix: ONGR\OXIDConnectorBundle\Entity
+                is_bundle: true
+            ONGRDemoOXIDBundle:
+                type: annotation
+                alias: DemoOXIDBundle
+                dir: Entity
+                prefix: ONGR\DemoOXIDBundle\Entity
+                is_bundle: true
+
+..
+
+.. code-block:: yaml
+
+    ongr_elasticsearch:
+        managers:
+            default:
+                mappings:
+                    - ONGROXIDConnectorBundle
+                    - ONGRDemoOXIDBundle
+..
+
+Then add configuration for bundles:
+
+.. code-block:: yaml
+
+    ongr_connections:
+        active_shop: oxid
+        shops:
+            oxid:
+                shop_id: 0
+        sync:
+            sync_storage:
+                mysql:
+                    connection: default
+                    table_name: ongr_sync_storage
+
+    ongr_oxid:
+        database_mapping:
+            oxid:
+                tags:
+                    @shop_tag: '_1'
+                    @lang_tag: '_1_en'
+                shop_id: 0
+                lang_id: 0
+        entity_namespace: ONGRDemoOXIDBundle
+
+..
+
+Then you will need to configure import and sync.
```

Comment:
From here, there is only bunch of services definitions, which probably for novice wouldn't say anything. Maybe we have some kind of a pipeline documentation somewhere, where it's explained what are we configuring in here and we can add link in here?